src/components/homePageCommonTasks/template.module.css

```diff
@@ -0,0 +1,24 @@
+.columnLeft {
+  background-color: white;
+}
+
+.columnRight {
+  background-color: var(--vads-color-primary-alt-lightest);
+}
+
+/* tablet breakpoint */
+@media (min-width: 40em) {
+  .wrapper {
+    background: linear-gradient(
+      to right,
+      white 0 50%,
+      var(--vads-color-primary-alt-lightest) 50% 100%
+    );
```

Comment:
I had first tried another strategy entirely but found that this gradient solution was actually the simplest and most effective. Where I deviate from the original strategy is that I drop this background for smaller screens and instead apply background colors to the content elements directly, which gives more accurate and reliable results.